app/backend/resources/schema.edn

```diff
@@ -1823,7 +1823,111 @@
+
+          {:db/ident :thk.contract/procurement-id
+           :db/valueType :db.type/string
+           :db/unique :db.unique/identity
+           :db/cardinality :db.cardinality/one
+           :db/doc "Procurement id, has to be unique as it is used in web links"}
+
+
+          {:db/ident :thk.contract/status
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Calculated contract status"}
+
+          ;; TA Regions
+          {:db/ident :ta.region/north
+           :thk/code "1800" :enum/attribute :ta/region}
+          {:db/ident :ta.region/east
+           :thk/code "1801" :enum/attribute :ta/region}
+          {:db/ident :ta.region/south
+           :thk/code "1802" :enum/attribute :ta/region}
+          {:db/ident :ta.region/west
+           :thk/code "1803" :enum/attribute :ta/region}
+          {:db/ident :ta.region/center
+           :thk/code "1804" :enum/attribute :ta/region}
+
+
+          {:db/ident :thk.contract/region
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "The region of the Project is displayed"}
+
+          ;; Contract types
+          {:db/ident :thk.contract.type/construction-works
+           :thk/code "3500" :enum/attribute :thk.contract/type}
+          {:db/ident :thk.contract.type/things
+           :thk/code "3501" :enum/attribute :thk.contract/type}
+          {:db/ident :thk.contract.type/services
+           :thk/code "3502" :enum/attribute :thk.contract/type}
+
+          {:db/ident :thk.contract/type
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "Comes from THK, not editable (activity_procurementType, activity_procurementTypeFK"}
+
+          {:db/ident :thk.contract/signed-at
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "Date when contract was signed, added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/deadline
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "The first deadline, added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/extended-deadline
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "The extended deadline, added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/warranty-period
+           :db/valueType :db.type/long
+           :db/cardinality :db.cardinality/one
+           :db/doc "The length of warranty period in months"}
+
+          {:db/ident :thk.contract/warranty-end-date
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "The end date of warranty period"}
+
+          {:db/ident :thk.contract/authority-contact
+           :db/valueType :db.type/string
```

Comment:
what does this string contain? a name or an email? should it be user ref?

Reply:
There is a doc for "authority-contact" that states "Added manually in the detail view of contract". So, it can be free-form text at the moment.

Reply:
Made it a ref as the safer option